pahfit/feature_strengths.py

```diff
@@ -103,6 +109,79 @@ def featcombine(ftable):
                          cfdic[feat]['range'][-1],
                          np.sum(dfs[mask]['strength']),
                          None,
-                         None])
+                         np.sum(dfs[mask]['eqw'])
+                         ])
 
     return cftable
+
+
+def eqws(comp_type, x_0, amp, fwhm_stddev, obs_fit):
+    """
+    Calculate the emission features equivalent width
+    (integral[(I_nu-I_cont)/I_cont d_lam]) in microns.
+
+    Parameters
+    ----------
+    comp_type : string
+        type of emission component (Drude1D/Gaussian)
+    x_0 : float
+        central wavelength of the feature.
+    amp : float
+        central intensity of the feature.
+    fwhm_stddev : float
+        fwhm or stddev of the feature depending on comp_type.
+    Returns
+    -------
+    eqw : float
+        the equivalent width of the feature
+    """
+    # Check if the emission component is Gaussian and calculate fwhm.
+    if comp_type == 'Gaussian1D':
+        fwhm = 2 * fwhm_stddev * np.sqrt(2 * np.log(2))
+    else:
+        fwhm = fwhm_stddev
+
+    # Get range and wavelength region for integration.
+    low = x_0 - (fwhm * 6)
+    lmin = low if low > 0 else 0.
+    lmax = x_0 + (fwhm * 6)
+    lam = np.arange(100) / 99 * (lmax - lmin) + lmin
```

Comment:
This line is slightly confusing to me. lam = np.linspace(lmin, max, num=100) would also produce the same result. I find np.linspace will be more intuitive than np.arange in this case.

Reply:
Agreed, it's the same thing. I will change it.